### ISSUE TITLE
[Adaptive Cards-FlightDetails.Json]: "Flight to JFK" is not defined as a heading on 'FlightDetails.Json' card.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9246]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen Reader: Narrator/NVDA

**Repro steps:**

1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the Adaptive Cards Visualizer application.
3. Navigate to FlightDetails.Json button by using Tab key and activate it.
4. FlightDetails adaptive card will be appeared on right sight of the pane.
5. Navigate to FlightDetails card by using Tab key.
6. Navigate in scan mode to "Flight to JFK" in heading mode by pressing "H" key and observe the issue.

**Actual Result:**
"Flight to JFK" is not defined as a heading on 'FlightDetails.Json' card. Screen reader is only announcing as, "Flight to JFK".

**Note**
Same issue is also repro with 'Passengers' as it is not defined as heading under 'Flightitinerary' card. (It should be defined inside the h3 heading)

**Expected Result:**
"Flight to JFK" should be defined under a heading. Screen reader should announce as, "Flight to JFK heading level 3".

**User Impact:**
It will be confusing & difficult for the users if Heading will not be defined inside the adaptive card.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/13778412-ea88-41af-a6f6-760292615bd0

![Image](https://github.com/user-attachments/assets/a4799ad3-bb59-4753-a1b8-772a67173ac6)

![Image](https://github.com/user-attachments/assets/92da5a4f-ec9e-451d-8be9-3f62413c4f91)

### Card JSON

```JSON
N/A
```

**Comment by Rishikesh6521** [comment]
Original Bug ID: #7618

**Comment by vagpt** [comment]
Since the product is getting deprecated, hence closing this issue.
